auth_mellon_handler.c

```diff
@@ -1791,8 +1791,8 @@ static int am_handle_reply_common(request_rec *r, LassoLogin *login,
     if (response->parent.Destination) {
         if (strcmp(response->parent.Destination, url)) {
             AM_LOG_RERROR(APLOG_MARK, APLOG_ERR, 0, r,
-                          "Invalid Destination on Response. Should be: %s",
-                          url);
+                          "Invalid Destination on Response. Should be: %s instead of %s",
```

Comment:
Could you rewrite the sentence so that it makes sense after this change? Keeping the colon makes the log line a bit strange. I'd suggest something like `Should be \"%s\", but was \"%s\".